### ISSUE TITLE
[RFC] merge pull request vs squash and merge

### ISSUE THREAD

**Issue by massich** [mne-tools/mne-python#5445]
Is there a way to make squash and merge the default behaviour when we press the GREEN button?

**Comment by agramfort** [comment]
It’s already the default afaik

**Comment by massich** [comment]
I don't think so. https://github.com/mne-tools/mne-python/commit/c6e62d6c11c557cd69ce0b4e88bb17b5ca8c47dd is a merge commit that it should not be there if I had taken more care when clicking the green button

**Comment by larsoner** [comment]
You can disable options, but I don't think we want to do that. IIRC it defaults to whatever you as a maintainer did in your previous green-button-click.
